var/spack/repos/builtin/packages/sopt/package.py

```diff
@@ -19,7 +19,10 @@ class Sopt(CMakePackage):
     maintainers("tkoskela", "mmcleod89", "20DM")
     license("GPL-2.0")
 
+    version("5.0.1", sha256="915000cf4d636b6830d6d37eb67e92a100001604ed30d34210232e3b583ba2a0")
```

Comment:
@alecbcs This version could not be verified, because the upstream author used a capital `V` in the version tag. CI wasn't failing though, which is an error.

Reply:
Thanks for catching the typo, I've created a `v5.0.1` tag upstream.